src/main/assets/scss/main.scss

```diff
@@ -1,14 +1,20 @@
 @import '~govuk-frontend';
 
-.nav-right {
-    float: right;
-    margin: -1.6vw 0px 0vh 0;
-    li {
-      a {
-        font-size: 19px;
-      }
-    }
+@media (min-width: 48.0625em) {
+  .govuk-header__content {
+    width: 39%;
+  }
+
+  .govuk-header__logo {
+     width: 60%;
```

Comment:
im not sure changing too much of the gov stylings is the best way to go, whats the logic for going against the gov styling here?

Reply:
it is because the title is long and it is displayed in 2 lines at the bottom of the logo instead of one line at the top aking the header too high.